### DISCUSSION TITLE
Python is not installed

### DISCUSSION THREAD

**Discussion by karadoulis** [Taxel/PlexTraktSync#1963] [Support] [answered]
Trying to run setup.bat and it is saying that I don't have python installed but it is already installed.
Any help would be much appreciate it.
Thanks in advance.

**Comment by karadoulis** [comment, marked as answer]
Solved it.
Solution: When installing python you should tick the two check marks on the bottom saying PATH and admin rights.

**Reply by glensc** [reply]
as guides says. describe your solution to help others.
